### PR TITLE
Initial proposal for JSON Schema blog guidelines

### PR DESCRIPTION
**Pull request by benjagm**
**Summary**:
Related: #474 
This PR provides an initial version of blog guidelines to better handle future external contributions.

Mostly inspired by CNCF Blog guidelines.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**
No

### CODE REVIEW

**Review by gregsdennis — COMMENTED** [review]
I think this is good, but does it address the issue of that PR?  It could be argued that the critism it contains is long-term beneficial for JSON Schema.  They're making recommendations for changing the spec because they feel that users would benefit from JSON Schema working a bit (albeit fundamentally) differently.

I don't think that's appropriate for a post on _our_ blog, though.  I think those kinds of things need to be realized as issues where we can discuss the benefits of the changes, and potentially (eventually) incorporate some of them.  Then, later, we can invite them to write a post about the full experience of recommending changes and seeing them through.

---

I also wonder if this needs to be in the blog repo instead of the community repo.

File: docs/blog-guidelines.md
Comment (jdesrosiers) [inline docs/blog-guidelines.md:51]:
```suggestion
Please submit your PR for consideration and we will start the PR process. If the article is not suitable for the JSON Schema blog, we will provide feedback and direction.
```

**Review by jdesrosiers — APPROVED** [review]
Found one typo, but otherwise it looks great.

**Comment by benjagm** [comment]
> does it address the issue of that PR?

I think the document has a good baseline for cases like that one in different ways:
1. We are restricting who can send blog posts with this sentence "The JSON Schema blog serves as a channel for project maintainers and contributors, implementation maintainers and JSON Schema Champions to share content with the JSON Schema Community." With this phrase we intend to express that the blog is from the community to the community and only relevant/trusted contributors of the community can publish here and in this case the authors are new joiners, they don't have that level of Community reputation.
2. We are restricting the types of blog posts. This is not directly restricting this type of content but it gives some soft arguments as well.
3. We are explicitly saying that there is review process and the article could be rejected with this sentence "If the article it’s not suitable for the JSON Schema blog, we will provide feedback and direction."
4. We introduce clarification that the content should be beneficial to the community with this sentence:  "Critical commentary or broad issues must be approached with sensitivity, professionalism, and tact in a way that is beneficial and positive for the Community." In the current case this argument IMHO is strong enough to take a decision to protect the JSON Schema Community benefit.

**Comment by benjagm** [comment]
> I also wonder if this needs to be in the blog repo instead of the community repo.

The blog repo will be soon deprecated as the blog is integrated with the site in the new website. I see 2 options here:

1. Locate the md file in the blog section in the new website repo.
2. keeping together these kind of contribution policies under the community repo. Some examples:
- https://github.com/cncf/foundation/blob/main/blog-guidelines.md
- https://github.com/falcosecurity/community/blob/main/blog-contribution-guidelines.md

**Comment by gregsdennis** [comment]
@benjagm if you think we're covered and we can say, "Thanks, but no.  Let's discuss other options elsewhere," then I'm happy.

If we want to keep all policy documents together, which I'm not opposed to (e.g. for housekeeping purposes), could we at least add a "guidelines"-ish file to the blog folder that simply links to the policy doc?

**Review by gregsdennis — APPROVED** [review]

**Comment by benjagm** [comment]
> if you think we're covered

I think we will be much better covered than before this and we will have way to improve this policy if required as new cases appear.

**Comment by benjagm** [comment]
> If we want to keep all policy documents together, which I'm not opposed to (e.g. for housekeeping purposes), could we at least add a "guidelines"-ish file to the blog folder that simply links to the policy doc?

100% Yes!

File: docs/blog-guidelines.md
Comment (Relequestual) [inline docs/blog-guidelines.md:43]:
```suggestion
Note: This is not a link to a JSON Schema website or from on the JSON Schema website.
```

Can you help me understand this sentence please? I'm not sure what it means 😅

File: docs/blog-guidelines.md
Comment (Relequestual) [inline docs/blog-guidelines.md:38]:
```suggestion
* Share contact info of speaker - twitter, slack, email
```

File: docs/blog-guidelines.md
Comment (Relequestual) [inline docs/blog-guidelines.md:45]:
```suggestion
The JSON Schema project will review the statement, as we do for the content for the blog, to approve the content. If changes are needed, we will provide feedback or suggested changes. 
```

**Review by Relequestual — CHANGES_REQUESTED** [review]
Overall, content looks great.
I've added a few small nits and questions.
(And I made a PR to the CNCF version of this document).

File: docs/blog-guidelines.md
Comment (benjagm) [reply docs/blog-guidelines.md:43]: Honestly I don't know what it means. I saw it in the blog guidelines I took as reference and kept it, but I think will be better to remove it.

**Comment by benjagm** [comment]
Thanks everyone for the reviews and comments. I tried to address all the changes with the last PR. As soon as we get this approved I'll work on adding the ish to the repos.

**Comment by benjagm** [comment]
> In fact, that sort of thing should probably be a CONTRIBUTING.md in the new website repo. May need some more general guidance, and then also the blog specific guidance.

I'll add a CONTRIBUTING.md to the new website repo before the new website roll-out.

**Comment by benjagm** [comment]
I realized that we don't have a definition of JSON Schema champion and that introduces some ambiguity in the expectations, this is why I just added a definition.

**Review by Relequestual — APPROVED** [review]
Thanks for your revisions

**Pull request merged**
